### PR TITLE
feat: homepage

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR
Create Hero component for homepage based on Figma. 

Need help with fine tuning styles @bhajneet

- [x] Download button for presenter
- [x] Fill white space with images? @bhajneet 

### Time spent on PR
- Saihaj: 2.5 hours 
- Bhajneet: ???

### Note
Based on #16 branch

### CODE REVIEW

**Comment by saihaj** [comment]
On mobile we should have these on new line 
079BF3DB-C572-4A93-986E-08398CE7E949

**Comment by bhajneet** [comment]
Fixed
Screen Shot 2021-08-15 at 15 01 06

**Comment by saihaj** [comment]
what pictures do we want to show for desktop @bhajneet ?

File: src/components/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    },
+  },
+  secondLink: {
+    marginLeft: '1rem',
+  },
+} )
+
+type HeroProps = {
+  title:string,
+  ctaList: LinkProps[],
```
Comment (saihaj) [inline src/components/Hero.tsx:47]: we should avoid abbreviations 
```suggestion
  callToActions: LinkProps[],
```

File: src/components/Section.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from 'react'
+import { createUseStyles } from 'react-jss'
+
+type SectionProps = {
+  children?: ReactNode,
```
Comment (saihaj) [inline src/components/Section.tsx:5]:
```suggestion
  children: ReactNode,
```

**Comment by bhajneet** [comment]
Why did you move the gif to the static folder? As you said, it's unoptimized by Gatsby

**Comment by saihaj** [comment]
We shouldn’t put it in pages directory

**Comment by bhajneet** [comment]
So where can you put it that would get optimized?

**Comment by saihaj** [comment]
They suggest importing directly
https://github.com/gatsbyjs/gatsby/issues/23598#issuecomment-621414940

**Comment by bhajneet** [comment]
Ok then, LGTM -- should we merge this?

**Comment by saihaj** [comment]
Do we have images for other 2 sections? Also need to get that download button working

**Comment by bhajneet** [comment]
I'll think about the images, but they're not required for this PR

**Comment by bhajneet** [comment]
Also, see Slack for the download button convo

**Comment by saihaj** [comment]
To be consistent we should avoid inline styles and use react-jss where possible

**Review by bhajneet — APPROVED** [review]

**Pull request merged**